src/main/java/com/kodilla/ecommercee/dto/Product.java

```diff
@@ -4,13 +4,80 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
-@AllArgsConstructor
-@NoArgsConstructor
-@Getter
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.validation.constraints.NotNull;
+
+//@AllArgsConstructor
+//@NoArgsConstructor
+//@Getter
+@Entity
+@Table(name = "product")
 public class Product {
+
     private  int id;
     private String name;
     private String description;
     private double price;
     private int groupId;
+
+    public Product() {
+    }
+
+    public Product(int id, String name, String description, double price, int groupId) {
+        this.id = id;
+        this.name = name;
+        this.description = description;
+        this.price = price;
+        this.groupId = groupId;
+    }
+
+    @Id
+    @NotNull
+    @Column(name = "id", unique = true)
```

Comment:
Na czacie wczesniej rozmawialismy ze moznaby zostawic nazewienictwo column dla hibernate. Takze annotacja @Column jest zbedna. I tak przy okazji z tego co mi wiadomo to przy id nie trzeba uzywac unique = true, poniewaz @Id jako PRIMIARY KEY jest juz z automatu ustawione na unique.